src/D2L.CodeStyle.Analyzers/D2L.CodeStyle.Analyzers.csproj

```diff
@@ -31,6 +31,7 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="DependencyScope\DependencyScopeAnalyzer.cs" />

```

Comment:
I'm going to be a jerk and come over and nit-pick about the name `DependencyScope` (the word scope) because I have some other analyzers in mind that I think could go into this category if the name is tweaked (and if they existed - there are some blockers but we have non-analyzers that check some properties of the graph that long-term could be awesome as analyzers.)